src/gui/transferlistwidget.cpp

```diff
@@ -393,6 +393,15 @@ void TransferListWidget::copySelectedNames() const
     qApp->clipboard()->setText(torrent_names.join("\n"));
 }
 
+void TransferListWidget::copySelectedHashes() const
+{
+    QStringList torrent_hashes;
+    foreach (BitTorrent::TorrentHandle *const torrent, getSelectedTorrents())
+        torrent_hashes << torrent->hash();
+
+    qApp->clipboard()->setText(torrent_hashes.join("\n"));
```

Comment:
so that (for Qt < 5.8) it will use `QStringList::join(QChar separator)` instead of `QStringList::join(const QString &separator)` in the sense of `QChar` is smaller than `QString`.